packages/wallet/src/operations/toKeyPair.ts

```diff
@@ -17,9 +17,15 @@ import { UNIVERSAL_WALLET_CONTEXT_URL, placeHolderImage } from '../constants';
 export const toKeyPair = async (
   mnemonic: string,
   index: number,
-  type = 'Ed25519'
+  type = 'Ed25519',
+  hdPath?: string
 ): Promise<any> => {
-  const keypair: any = await sidetreeCrypto.toKeyPair(mnemonic, index, type);
+  const keypair: any = await sidetreeCrypto.toKeyPair(
+    mnemonic,
+    index,
```

Comment:
index and hdPath are redundant.

`Ethereum — m'/44’/${assumedCoinType}’/0’/0/${index}`... vs .... `m'/44’/60’/0’/0/0`.

I think this ambiguity, redundancy should be removed, can you create an issue on this repo to track resolving it?

Reply:
I expect it will be expensive to fix, lots of places where this is used.

Reply:
created it here https://github.com/transmute-industries/sidetree.js/issues/340